src/app/clients/clients.component.html

```diff
@@ -2,7 +2,7 @@
   <div fxLayout="row" fxLayoutAlign="start center">
```

Comment:
@BLasan check the latest code for centers view https://github.com/openMF/web-app/pull/506/files, implement search feature in a similar manner, also can you rebase this pull request on latest code, it will be easy for me and you to compare.